src/utils.js

```diff
@@ -397,7 +413,11 @@ export function isCompatible(bcd, path, minVersion, application) {
   }
   // API namespace may be undocumented or not implemented, ignore in that case.
   if (api.__compat) {
-    const versionAdded = api.__compat.support[application].version_added;
+    const supportInfo = api.__compat.support[application];
+    const versionAdded = Array.isArray(supportInfo)
+      ? firstStableVersion(supportInfo)
+      : !Object.prototype.hasOwnProperty.call(supportInfo, 'flags') &&
```

Comment:
couldn't just turn supportInfo into an array of a single element inside `firstStableVersion` itself (if it isn't already an array) and just call `const versionAdded = firstStableVersion(supportInfo);` here?